src/Kmd.Logic.Gateway.Automation/Publish.cs

```diff
@@ -70,9 +73,23 @@ public async Task<IEnumerable<PublishResult>> ProcessAsync(string folderPath)
             }
 
             var apiPreValidation = new ApiPreValidation(folderPath);
-            if (!(await apiPreValidation.ValidateAsync(gatewayDetails).ConfigureAwait(false)))
+            var productPreValidation = new ProductPreValidation(folderPath);
```

Comment:
Why this created again, below collection this has been created

Reply:
Removed